test/custom/AuctionCanceled.test.js

```diff
@@ -18,7 +18,8 @@ const { MARKETS } = require('../helpers/constants');
 const { testCases } = require('../testCases.json');
 
 describe('Auction Canceled', async () => {
-    MARKETS.map((marketName) => {
+    // Noun auctions cannot be cancelled
+    MARKETS.filter(m => m !== MARKET_NAMES.NOUNS).map((marketName) => {
```

Comment:
fix: need to import `MARKET_NAMES`

Reply:
(addressed this but can't push)

Reply:
https://github.com/PartyDAO/partybid/pull/43/commits/7c246cc5dcbc97d4881719fd7f87ee879cab95d6